autokeras/utils.py

```diff
@@ -171,10 +170,8 @@ def validate_xy(x_train, y_train):
 
 def read_csv_file(csv_file_path):
     """Read the csv file and returns two separate list containing file names and their labels.
-
```

Comment:
according to the style guide, these line should be kept.

Reply:
Fixed the docstring style